docs/06-contributors/999-rfcs/000-stories/story-13.md

```diff
@@ -0,0 +1,169 @@
+# User Story 13 - Task List with uuid, Json and duration
+
+> **Status**: Expected released on 2023/02/16
+
+The following code is an iteration of Wing's task list.
+This version of it includes the following functional changes:
+- Using npm's uuid package inflight instead of the counter resource 
+- Storing Json instead of a string
+- Using duration for estimated effort
+
+Also, it includes some non-functional requirements: 
+- The Bucket should be replicated across different regions 
+  - +100 if we are able to do the same for azure as well
+- Least privileged permissions are granted
+- The code should work on sim, aws, azure
+- VScode should be able to autocomplete 
+- Console should be able to show all resources, and interact with them
+
+
+
+```ts (wing)
+bring cloud;
+// need to run: npm install uuid --save
```

Comment:
I think we will need a jsii wrapper for uuid in order for this to work.

```suggestion
// need to run: npm install @winglibs/uuid --save
```

Reply:
Life is tuff

Reply:
is uuid stuff important enough to include in the stdlib?

Reply:
I actually think it is important enough to include in the stdlib at some point, but for the sake of this story, @ekeren wanted some simple external library, and this is good enough I believe.

Reply:
Do we want to use the`@winglibs` namespace  for this?

Reply:
We don't have to. We can just publish something like wing-uuid for now